sos/report/plugins/rhui.py

```diff
@@ -27,15 +27,23 @@ def setup(self):
             "/var/log/rhui-subscription-sync.log",
             "/var/cache/rhui/*",
             "/root/.rhui/*",
+            "/etc/ansible/facts.d/rhui_*.fact",
         ])
         # skip collecting certificate keys
-        self.add_forbidden_path("/etc/pki/rhui/*.key")
+        self.add_forbidden_path([
+            "/etc/pki/rhui/*.key",
+            "/etc/pki/rhui/*/*.key",
+            "/etc/pki/rhui/*/*/*.key",
```

Comment:
Should we instead be modifying `add_forbidden_path` to recurse into directories with `glob`?

If we pass `recursive=True` to `glob` [here](https://github.com/sosreport/sos/blob/master/sos/report/plugins/__init__.py#L1205), then this could be:

```python
self.add_forbidden_path("/etc/pki/rhui/**/*.key")
```

Which would match all of this in addition to any potential new subdirs that may be added by RHUI later.

Reply:
Good point, I will update the PR accordingly.